<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_Colaboradores_Editing</class>
 <widget class="QDialog" name="Fm_Colaboradores_Editing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lbl_name">
           <property name="text">
            <string>Nome:</string>
           </property>
           <property name="buddy">
            <cstring>le_name</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_name"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_tel">
           <property name="text">
            <string>Telefone:</string>
           </property>
           <property name="buddy">
            <cstring>le_tel</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_tel"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lbl_username">
           <property name="text">
            <string>Username:</string>
           </property>
           <property name="buddy">
            <cstring>le_username</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_username"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lbl_password">
           <property name="text">
            <string>Senha:</string>
           </property>
           <property name="buddy">
            <cstring>le_password</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_password"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="lbl_accessType">
         <property name="text">
          <string>Tipo de Acesso:</string>
         </property>
         <property name="buddy">
          <cstring>cb_accessType</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_accessType">
         <item>
          <property name="text">
           <string>Selecione</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btt_update">
         <property name="text">
          <string>Atualizar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btt_cancel">
         <property name="text">
          <string>Cancelar</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
